### PR TITLE
Vijaya work

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

**Review by Richa199 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

File: html/VijayaAssignment4/question1.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<center>
+<div  style="background-color:lightslategrey; width:600px; height:300px; padding-top:25px;">
+<TABLE border="1" style="background-color:#f1f1f1;">
+<CAPTION><EM>A test table with merged cells</EM></CAPTION>
+<TR><TH rowspan="2"><TH colspan="2">Average
+    <TH rowspan="2">Red<BR>eyes
+<TR><TH>height<TH>weight
+<TR><TH>Males<TD>1.9<TD>0.003<TD>40%
+<TR><TH>Females<TD>1.7<TD>0.002<TD>43%
+</TABLE>
```
Comment (shubhamdalal49) [inline html/VijayaAssignment4/question1.html]: use small letter to make code better

**Review by urmilagagare — APPROVED** [review]

File: html/VijayaAssignment4/question1.html (hunk 1 above)
Comment (vijaya1811) [reply html/VijayaAssignment4/question1.html]: ok

**Review by vijaya1811 — COMMENTED** [review]
yes I done  changes

File: html/VijayaAssignment4/question1.html (hunk 1 above)
Comment (vijaya1811) [reply html/VijayaAssignment4/question1.html]: I done changes

**Pull request closed without merging**